include/aspect/material_model/melt_model/melt_simple_fraction.h

```diff
@@ -0,0 +1,102 @@
+/*
+  Copyright (C) 2015 - 2022 by the authors of the ASPECT code.
+
+  This file is part of ASPECT.
+
+  ASPECT is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2, or (at your option)
+  any later version.
+
+  ASPECT is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with ASPECT; see the file LICENSE.  If not see
+  <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef _aspect_material_model_melt_model_melt_simple_fraction_h
+#define _aspect_material_model_melt_model_melt_simple_fraction_h
+
+#include <aspect/material_model/interface.h>
+#include <aspect/simulator_access.h>
+#include <aspect/postprocess/melt_statistics.h>
+#include <aspect/melt.h>
+
+namespace aspect
+{
+  namespace MaterialModel
+  {
+    using namespace dealii;
+
+    namespace MeltModel
+    {
+      template <int dim>
+      class MeltSimpleFraction : public ::aspect::SimulatorAccess<dim>
```

Comment:
My initial vote is to make this more descriptive, and have it names something like `Katz03MantleMelting` or something equally descriptive such that in can be distinguished from future distinct melting models (and other types of reactions) that are added.